lxc/launch.go

```diff
@@ -4,45 +4,68 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/codegangsta/cli"
```

Comment:
Insert a new line after this import to keep it separate from the github.com/lxc/lxd/* ones.